etna/articles/tests/test_similar_items.py

```diff
@@ -74,7 +74,7 @@ def test_similar_items_ranking(self):
         # Items should be in 'best match' order
         # No draft items should be included
         test_page = ArticlePage.objects.get(id=self.original_page.id)
-        with self.assertNumQueries(4):
+        with self.assertNumQueries(10):
```

Comment:
This test failure should be interpreted as a sign that the implementation has missed the mark a little.